features/steps/s4s.py

```diff
@@ -28,6 +37,61 @@ def parse_mu_ccds_mapping(mu_ccds):
 register_type(MU_CCDS=parse_mu_ccds_mapping)
 
 
+@given('I have a valid conformance statement')
+def step_impl(context):
+    assert context.conformance is not None, \
```

Comment:
This statement only gets run in `10-sync-for-science.feature`. All the actual resource tests use `this server supports {resource_type}` which has the early escape of:

```
# If there is no conformance statement, assume this resource is supported
if context.conformance is None:
    return
```